modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java

```diff
@@ -1028,7 +1028,8 @@ else if (devDependencies.containsKey("liferay-npm-scripts")) {
 			return false;
 		}
 
-		VersionNumber versionNumber = VersionNumber.parse(dependencyName);
+		VersionNumber versionNumber = VersionNumber.parse(
+			(String)devDependencies.get(dependencyName));
```

Comment:
I feel so ashamed. Thanks for fixing!

![ashamed](https://user-images.githubusercontent.com/7074/94719470-7bda3900-0353-11eb-9769-773ee6d69b14.gif)